src/Symfony/Component/Config/Definition/Builder/NodeDefinition.php

```diff
@@ -76,6 +77,30 @@ public function example(string|array $example): static
         return $this->attribute('example', $example);
     }
 
+    /**
+     * Sets the documentation URI, as usually put in the "@see" tag of a doc block. This
+     * can either be a URL or a file path. You can use the placeholders {package},
+     * {version:major}, {version:minor}, and {version:patch} in the URI.
+     *
+     * @return $this
+     */
+    public function documentationUri(string $uri, ?string $package = null): static
+    {
+        try {
+            preg_match('/\d+.\d+.\d+/', InstalledVersions::getVersion($package) ?? '', $m);
```

Comment:
This should not be done if `$package` is `null`

Reply:
Indeed